packages/babel-traverse/src/path/family.ts

```diff
@@ -183,11 +183,10 @@ function getStatementListCompletion(
         break;
       }
       if (i === paths.length - 1) {
-        Array.prototype.push.apply(completions, statementCompletions);
+        completions.push(...statementCompletions);
       } else {
-        Array.prototype.push.apply(
-          completions,
-          statementCompletions.filter(c => c.type === BREAK_COMPLETION),
+        completions.push(
+          ...statementCompletions.filter(c => c.type === BREAK_COMPLETION),
         );
         lastNormalCompletions = statementCompletions.filter(
           c => c.type === NORMAL_COMPLETION,
```

Comment:
Would it make sense to do this with a for loop, instead of filtering statementCompletions twice?